### DISCUSSION TITLE
Two Sensor Icons - Per Each Home Page Entity

### DISCUSSION THREAD

**Discussion by indyjonesuk** [Blackymas/NSPanel_HA_Blueprint#720] [Ideas]
Hi,

Thank you for the fantastic work.
Where my panel is positioned it is outside the bathroom (in the bedroom), so displays & controls both areas.
What would be great for me is for to display two icons for a entity e.g. Bedroom & Temperature Icons. I tried it for the main temperature but it only had space for for the first icon.

Thank you!

**Comment by edwardtfn** [comment]
I'm not sure if I got what you have in your mind.
Could you please create a Enhancement request in the [Issues](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) adding a bit more of details?
